src/cps/env.cpp

```diff
@@ -1,12 +1,13 @@
 // Copyright © 2023-2024 Dylan Baker
-// Copyright © 2024 Tyler Weaver
+// Copyright © 2024 Dylan Baker
```

Comment:
Do you have an editor tool doing this, your name is in the first line.

Reply:
Yes, my editor is doing that and I missed undoing one of them apparently... Sigh

Reply:
np, figured it was some tool doing this